### ISSUE TITLE
Tag dockerized version of Chloes code prior to "open heart surgery."

### ISSUE THREAD

**Issue by amycoffin** [BRCAChallenge/brca-exchange#830]
Before we start making major changes, we'd like to assign a GitHub tag to version 1 of the Probabilities code so that we can easily access a version that runs. We might as well tag the dockerized version rather than the original one

**Comment by rcurrie** [comment]
@amycoffin the code has not been merged into BRCA yet (pull request is #819). As its as written likely wrong I'm not sure we should integrate it back in net of the dockerization and local ensembl additions.

For posterity here is the version of my forked repo that's fully tested before we do the big change to getVarConsequences():

https://github.com/rcurrie/brca-exchange/tree/44eaee3f42f97cbc09ea31dcc15867fcbf24f4de

**Comment by rcurrie** [comment]
Closing - tag in the comment above and switching to a single branch with docker and getVarConsequences() after we validate.
